components/project-loader/project-loader.jsx

```diff
@@ -54,6 +55,11 @@ export default React.createClass({
       return Object.assign(combinedParams, { page: 1 });
     }
 
+    if (combinedParams.moderationState) {
+      combinedParams.moderationstate = combinedParams.moderationState.label;
```

Comment:
Ummm key in `combinedParams` will later be passed as query params to the `/entry/?` call. `moderationstate` is the query param the API understands so that's why I didn't give it another (better) name. If it helps I will drop an inline note there.